nn_sdwan/manifest.json

```
@@ -0,0 +1,41 @@
+{
+  "display_name": "Netnology SDWAN",
+  "maintainer": "info@netnology.io",
+  "manifest_version": "1.0.0",
+  "name": "nn_sdwan",
+  "metric_prefix": "nn_sdwan.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "SDWAN Controller Metric Exporter by Netnology",
+  "guid": "a7c79ba9-a4b7-4fa4-b4aa-3868458e2101",
+  "support": "extras",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Netnology SDWAN Integration",
+  "categories": [
+    "monitoring",
+    "network"
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "nn-sdwan",
+  "assets": {
+    "configuration": {
+      "spec": "assets/configuration/spec.yaml"
+    },
+    "dashboards": {
+      "Netnology SDWAN Overview": "assets/dashboards/overview.json"
```

This should either be:
```suggestion
      "Netnology SDWAN Overview": "assets/dashboards/nn_sdwan_overview.json"
```
or you'd have to rename the json in assets/dashboard to `overview.json`